examples/dll-app-and-vendor/webpack.config.js

```diff
@@ -0,0 +1,15 @@
+var fs = require("fs");
+
+if(!fs.existsSync("./js")) {
+	// build the vendor first,
+	// since automated integration tests
+	// seem to run the webpack config not the build.js
+	require("./build");
```

Comment:
@sokra doesn't it make things more confusing have to go to multiple folders for a single example? That would mean 6 folders for 3 examples.